components/atomic/buttons/ButtonControlGroup/ButtonControlGroup.tsx

```diff
@@ -29,6 +31,15 @@ function ButtonControlGroup({
       >
         {children}
       </ButtonControlDrawer>
+    ) : "modalLabel" in props ? (
```

Comment:
Hmm... to make room for other modals, you could add a `modalConfirm` boolean flag, instead of checking for `modalLabel`. Then `modalLabel` and `modalBody` can be reused in other modal button patterns. What do you think, @zdavis ?